### DISCUSSION TITLE
Is it reasonable to assume that hass-chargeamps also supports Charge Amps Luna?

### DISCUSSION THREAD

**Discussion by HaunsTM** [hass-chargeamps/hass-chargeamps#71] [Q&A] [answered]
Is it reasonable to assume that hass-chargeamps also supports https://www.chargeamps.com/product/charge-amps-luna/ (I haven't gotten the hardware yet)

**Comment by jschlyter** [comment, marked as answer]
Yes, I believe so, but I have not tested.

**Reply by HaunsTM** [reply]
Then I might be a tester for you. Just need to figure out if it is possible to manage the device completely local or if I have to deal with the retailers cloud app (which I don't want to do).
